libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorMapper.kt

```diff
@@ -71,6 +71,23 @@ internal class NavigatorMapper {
         return status.getRouteProgress(directionsRoute, routeBufferGeoJson, remainingWaypoints)
     }
 
+    fun toIncidentInfo(info: com.mapbox.navigator.IncidentInfo?): IncidentInfo? =
```

Comment:
why did you replace the previous implementation `private fun com.mapbox.navigator.IncidentInfo.toIncidentInfo(): IncidentInfo?`? We can just make it public

Reply:
just make it public won't help because it would be a public extension fun inside `Mapper` object. AFAIR we can't call it for values outside the `Mapper`.
to make git diff cleaner I reverted changes and added public fun which calls private extension under the hood.
```kotlin
fun getIncidentInfo(info: com.mapbox.navigator.IncidentInfo?) = info?.toIncidentInfo()
```

Reply:
wow, I've got that Mapper is actually class. Interesting